docs/source/Af1.rst

```diff
@@ -10,79 +10,147 @@ Basic data access
 .. autosummary::
     :toctree: generated/
 
-    client_location
-    config
-    lookup_release
-    open_file
-    read_files
     releases
-    results_cache_get
-    results_cache_set
     sample_sets
+    lookup_release
+    lookup_study
```

Comment:
Cutting down to functions that are likely to be more interesting and useful for most users.